File: Tasks/VideoDisplay.cpp
#include <iostream>
#include <opencv2/highgui.hpp>

using namespace cv;

int main() {
    VideoCapture video(0);

    if(!video.isOpened()) {return -1;}

    Mat frame;

    while(video.read(frame)) {
        imshow("Video stream", frame);
        char c = waitKey(1);
        if (c == 'q' || c == 'Q') {
            break;
        }
    }
}

File: Tasks/VideoDisplay.py
import numpy as np
import cv2


cap = cv2.VideoCapture(0)

while True:
    ret, frame = cap.read()

    cv2.imshow('video stream', frame)
    if cv2.waitKey(1) == ord('q') or cv2.waitKey(1) == ord('Q'):
        break

cap.release()
cv2.destroyAllWindows()